index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+    <head> 
+        <title>Kalina Gardziała - CV</title>
+        <link rel="stylesheet" href="style.css" /> 
+    </head>
+     <body>
+        <div class="container">
+            <header> 
+                <img src="ja.jpg" alt= "My Profile Picture"/> 
```

Comment:
Always use English words for files, folders, classes etc..  So it should be me.jpg